### PR TITLE
[fpga] Update documentation based on #8138 analysis

### PR DESCRIPTION
**Pull request by tjaychen**
Signed-off-by: Timothy Chen <timothytim@google.com>

### CODE REVIEW

**Comment by tjaychen** [comment]
@vogelpi did such a thorough job illustrating the issue in #8138 i was hoping to capture it somewhere for future reference.
I'm sure this won't be the last time we run into congestion issues related to this.

File: doc/ug/getting_started_fpga.md [hunk 1]
```diff
@@ -61,6 +61,12 @@ The `fileset_top` flag used above is specific to the OpenTitan project to select
 The resulting bitstream is located at `build/lowrisc_systems_chip_earlgrey_cw310_0.1/synth-vivado/lowrisc_systems_chip_earlgrey_cw310_0.1.bit`.
 See the [reference manual]({{< relref "ref_manual_fpga.md" >}}) for more information.
 
+### Dealing with FPGA Congestion Issues
+
+The default vivado tool placement may sometimes result in congested FPGA floorplans.
```
Comment (vogelpi) [inline doc/ug/getting_started_fpga.md:66]: You may want to say that the congestion can lead to unpredictable and long implementation times and results. This is what the user might experience and how he/she realizes that something is going wrong.

Comment (vogelpi) [reply doc/ug/getting_started_fpga.md:66]: Nit: want to capitalize Vivado?

**Review by vogelpi — APPROVED** [review]
This looks good, thanks. I feel really honored that you want to preserve my explanation Tim. But I would like to emphasize that you did the hard work of tracking down the problem and in the end also solved it! Thanks again for that!

**Comment by tjaychen** [comment]
> This looks good, thanks. I feel really honored that you want to preserve my explanation Tim. But I would like to emphasize that you did the hard work of tracking down the problem and in the end also solved it! Thanks again for that!

no thank you for digging through it. I kind of made a guess that this was the issue but didn't dig this deeply to verify.  Seeing the physical evidence to back this up really helps explains things.

File: doc/ug/getting_started_fpga.md (hunk 1 above)
Comment (tjaychen) [reply doc/ug/getting_started_fpga.md]: updated!

**Review by imphil — APPROVED** [review]

**Pull request merged**